src/main/java/de/tum/in/www1/artemis/repository/StatisticsRepository.java

```diff
@@ -71,6 +115,15 @@
             """)
     List<Map<String, Object>> getConductedExams(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate);
 
+    @Query("""
+            select e.endDate as day, count(e.id) as amount
+            from Exam e
+            where e.endDate >= :#{#startDate} and e.endDate <= :#{#endDate} and e.course.id = :#{#courseId}
+            group by e.endDate
+            order by e.endDate asc
+            """)
+    List<Map<String, Object>> getConductedExamsForCourse(@Param("startDate") ZonedDateTime startDate, @Param("endDate") ZonedDateTime endDate, @Param("courseId") Long courseId);
```

Comment:
please use a custom object here and not a map

Reply:
Done